docs/griptape-framework/drivers/vector-store-drivers.md

```diff
@@ -49,6 +49,30 @@ print("\n\n".join(values))
 
 ```
 
+### Griptape Cloud Knowledge Base
+
+The [GriptapeCloudKnowledgeBaseVectorStoreDriver](../../reference/griptape/drivers/vector/griptape_cloud_knowledge_base_vector_store_driver.md) can be used to query data from a Griptape Cloud Knowledge Base. Loading into Knowledge Bases is not support, only querying. Here is a complete example of how the driver can be used to query an existing Knowledge Base:
```

Comment:
`is not support` -> `is not supported at this time`.

`driver` -> `Driver`

Reply:
Do we really want Driver capitalized, it isn't for rest of entries (can update all if we want)